Accessibility UI

chloeylee opened the pull request:
[Accessibility Audit Report](https://docs.google.com/document/d/1DMITK24JleP-40urFFJ0pUOUmR_JDLG2Fv0AvcuJfc8/edit?usp=sharing)
- Made website compatible with mobile devices/varying screen sizes
- Resolved all contrast and semantic errors (i.e. heading level errors)
- Made website more compatible with assistive technologies by adding proper form labels in the filtering section of the data visualizations

joelsjlee commented:
I'm also adding an updating of the JQA data, moving the filters around a bit such that they have less nodes and edges, in order to improve efficiency of the visualization. Now the ranges of nodes for each time period are from 127 to 167. Whereas the current range is 167 to 278. You should checkout my branch of this PR, and start a local server to compare with the AWS website. Let me know what you think.

chloeylee committed:
almost all contrast errors fixed

filtering section in geo reference map still has contrast errors

chloeylee committed:
bookshelf is screen resposive

also slightly changed the styling of the container holding the .button.card

chloeylee committed:
navbar menu is screen size responsive

changes made
- on mobile screens, navbar menu list items stack vertically
- on hover, dropdown options appear below the list option and displace any list items below it (so that all options are still visible when hovering a dropdown)
- dropdown list items are italicized so that they can be differentiated

chloeylee committed:
global css to make website responsive to screen size

chloeylee committed:
filter section contrast errors resolved

styling is in multiselect-dropdown.js and global.css

chloeylee committed:
fixes for coref page

- filtering system looks better (sliders)
- still need to fix missing/orphaned labels in dropdown menu (community filter)

chloeylee committed:
missing alt text and most heading error fixed

chloeylee committed:
alt texts added for internal landing pages

- TODO: change the projects internal landing images to full names rather than initials

chloeylee committed:
heading level errors fixed

chloeylee committed:
missing/orphaned form label error fixed

in co-reference graph filtering dropdown menu

chloeylee committed:
typo fixed

chloeylee committed:
bookshelf only shows author/tools if data exists accordingly

currently just hardcoded to the author initials
- recommend a better system for this in the future

chloeylee committed:
mobile device compatibility fixes

chloeylee committed:
timeline is fixed!

includes screen width adjustment/rerendering

chloeylee committed:
mobile device compatibility cont.

- hopefully fixed weird spacing for the navbar and footer
- check if button-1 objects have text overflowing
